docs-site/docs/guides/transactions.md

```diff
@@ -0,0 +1,178 @@
+
+Now we have a complete picture of the transaction process from the point of **opening a wallet** to **submitting a transaction**. This allows us to [create a Funnel in 0xArc App](/guides/custom-events#custom-events-and-funnels) to answer our question above.
+
+We can add additional events between the trigger and submission to capture more information as desired. See the [Custom Events Guide](/guides/custom-events) for more info.
+
+### React Example
+
+```tsx
+import { useEffect } from 'react'
+import { useArcxAnalytics } from '@0xarc-io/analytics'
+import { useWeb3React } from '@web3-react/core'
+import { injected } from './web3Provider'
+
+const Component = () => {
+  const sdk = useArcxAnalytics()
+  const { account, chainId, activate, library } = useWeb3React()
+
+  // 1. Track wallet connections
+  useEffect(() => {
+    if (account && chainId) {
+      sdk.wallet({ account, chainId })
+    }
+  }, [account, chainId, sdk])
+
+  // 2. Track when the chain changes
+  useEffect(() => {
+    if (sdk && chainId) {
+      // If the chain ID is changed, we emit the chain changed event
+      sdk.chain({ chainId })
+    }
+  }, [chainId, sdk])
+
+  const connectWallet = async () => {
+    // This function call opens the MetaMask wallet,
+    // triggering the useEffect below once the user connects
+    await activate(injected)
+  }
+
+  // Example: Simulating a transaction call and tracking when the transaction was triggered and signed
+  const onSwapClicked = async () => {
+    const signer = library.getSigner(account)
+
+    // 3. Track a custom event when the swap starts
+    sdk.event('SWAP_STARTED', {
+      from: account,
+      to: '0x...',
+      amount: 42,
+    })
+
+    const tx = await signer.sendTransaction({ to: '0x...', value: 42 })
+
+    // 4. Track the signed transaction
+    sdk.transaction({
+      transactionHash: tx.hash,
+      account,
+      chainId,
+      metadata: {
+        action: 'SWAP',
+      },
+    })
+  }
+
+  return (
+    <div>
+      <button onClick={connectWallet}>Connect Wallet</button>
+      <button onClick={onSwapClicked}>Swap</button>
+    </div>
+  )
+}
+```
+
+As you can see in the example above, we also show how triggering the `chain change` event look in practice.
+
+---
+
+## Confirming a Submitted Transaction
+
+Assuming you've sent the required `.transaction()` event as earlier outlined, 0xArc will automatically match the transaction hash of the transaction event to the submitted transaction on the blockchain, and confirm if a transaction was completed or not.
+
+This is possible since 0xArc pre-indexes all transaction data from the blockchain and matches the transaction hashes of the transaction events to the blockchain data.
+
+Therefore, you do not need to manually track the transaction hash of a submitted transaction, as 0xArc will automatically do this for you.
```

Comment:
This line is confusing. I suggest removing it because it's not clear how the user *could* manually track the transaction hash of a submitted transaction if they already use the `.transaction()` function.
```suggestion
```